server/lib/analytics/index.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * External dependencies
+ */
+import superagent from 'superagent';
+
+/**
+ * Internal dependencies
+ */
+import config from 'config';
+import { statsdUrl } from '../../../client/lib/analytics/statsd';
```

Comment:
@ehg Unfortunately it would appear that it is not that simple. 🙁 

It looks like`lib/analytics` would technically work, but only because there is no module `statsd` inside of `server/lib/analytics`. If at some point such a module is created, it will be loaded instead of the `client` version, since `server` appears before `client` in `NODE_PATH`.

On the topic of `NODE_PATH`, we could almost use `client/lib/analytics/statsd`, except that there are a few different places where `NODE_PATH` is set, and they are not consistent 😄 

For example, in the various `test` scripts in `package.json`, the `.` directory is not added to `NODE_PATH`. Similarly in the `Dockerfile`. So using `import ... from 'client/...'` might work in development, but it currently wouldn't work anywhere else, AFAICT.

I wonder: would it make sense to have a better place for common code instead? This `statsd` file is used by both the client and the server, and so it really doesn't make much sense for it to live in `client`. But I wasn't sure if there was a better place?